big_tests/tests/mongoose_user_stats_SUITE.erl

```diff
@@ -30,6 +30,7 @@ suite() ->
 all() ->
     [
         user_stats_are_reported_to_google_analytics_when_mim_starts
+        , all_clustered_mongooses_report_the_same_client_id
```

Comment:
In new code we don't start a line with `,`, please move the `,` to the prev line.

Reply:
I do not insist on this, but I recently came across a slightly different formatting style:
https://kowainik.github.io/posts/2019-02-06-style-guide#alignment
I know it is Haskell, not Erlang, but I like the idea as you can just comment a single line and the whole list will still compile, which is not the case if you comment out first or last one.
So if you have 3 tests and want to comment out the last one or 2 last of them:
```erlang
all() ->
[
  test1
%  , test2
%  , test3
]
```